server/controllers/medical/patient.js

```diff
@@ -192,7 +198,13 @@ function handleFetchPatient(uuid) {
     'ON p.debitor_uuid = d.uuid AND d.group_uuid = dg.uuid AND p.project_id = proj.id ' +
     'WHERE p.uuid = ?';
 
-  return db.exec(patientDetailQuery, [uuid]);
+  return db.exec(patientDetailQuery, uuid)
```

Comment:
I began first of all to try adding new interfaces, but I notices that every needed interfaces were present, so I continued directly implementing the cash box component.
This change was made at the very beginning.